### DISCUSSION TITLE
Mapping <S-Esc>?

### DISCUSSION THREAD

**Discussion by huyz** [macvim-dev/macvim#1341] [Q&A] [answered]
Is anyone else able to map `<S-Esc>` in the GUI?

It works in neovim and it seems to work in gvim: https://github.com/vim/vim/issues/2520#issuecomment-356133334

For me, I just get the same thing as `<Esc>`.

In iTerm, I able to get a proper `^[[27;2;27~` from vim, which I can work with.

**Comment by ychin** [comment, marked as answer]
Seems to work just fine? How does it not work? Can you be more specific?

**Reply by huyz** [reply]
Oh really?
Inside MacVim GUI, in insert mode, if I hit `<C-S-v>` and then `<S-Esc>`,  I just get `^[` instead of `<S-Esc>`.
Do you not get that?

**Reply by ychin** [reply]
Have you actually tried mapping it? I think that's just how it's displayed.

**Reply by huyz** [reply]
I thought I had.  But apparently I didn't cause it works without interfering with insert-mode `<Esc>` this time. 

Sorry for wasting your time.
